<launch>

    <!-- location node -->
    <node pkg="mecanumLoc" type="mecanumEnc" name="localization"/>

    <!-- Navigation node -->
    <node pkg="nav_mec" type="navMec_node" name="navMec" output="screen"/>

</launch>